### PR TITLE
update transition-guide.md

### PR DESCRIPTION
**Pull request by binh-ampere**
update transition-guide.md for Ampere processors

### CODE REVIEW

File: transition-guide.md [hunk 1]
```diff
@@ -18,29 +21,29 @@ The following transition guide is organized into a logical sequence of steps as
     * Step 5 - Testing and optimizing your workload
     * Step 6 - Performance testing
 * [Infrastructure and deployment](#infrastructure-and-deployment)
-    * Step 7 - Update your infrastructure as code
```
Comment (dneary) [inline transition-guide.md]: I think this line should be left as-is. It's referring to Terraform, Pulumi, etc.

File: transition-guide.md [hunk 2]
```diff
@@ -72,60 +74,63 @@ To transition and test your application, you will need a suitable Graviton envir
...
 
 Important: This repository has sections dedicated to [Optimization](optimizing.md) and a [Performance Runbook](perfrunbook/graviton_perfrunbook.md) for you to follow during this stage.
 
-If after reading the documentation in this repository and following the recommendations you do not observe expected performance then please reach out to your AWS account team, or send email to [ec2-arm-dev-feedback@amazon.com](mailto:ec2-arm-dev-feedback@amazon.com) with details so we can assist you with your performance observations.
+If after reading the documentation in this repository and following the recommendations you do not observe expected performance then please reach out to your Ampere account team, or send email to [support@amperecomputing.com](mailto:support@amperecomputing.com) with details so we can assist you with your performance observations.
 
 
 ### Infrastructure and deployment
 
-**Step 7 - Update your infrastructure as code**
```
Comment (dneary) [inline transition-guide.md:104]: Same comment here - I think this is better unchanged.

File: transition-guide.md [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
+With the transition from legacy x86 to AArch64 and particularly Ampere processors gaining momentum, developers are looking for ways to expedite the migration for existing codebases. Ampere processors available on both bare-metals and instances on many major Cloud Service Providers.
```
Comment (dneary) [inline transition-guide.md]: The line you're changing came from me - could you try to merge your content with mine?

File: transition-guide.md [hunk 4]
```diff
@@ -72,60 +74,63 @@ To transition and test your application, you will need a suitable Graviton envir
...
 
 
 ### Infrastructure and deployment
 
-**Step 7 - Update your infrastructure as code**
+**Step 7 - Apply Arm64 code to infrastructure**
 
-Now you have a tested and performant application, its time to update your infrastructure as code to add support for Graviton-based instances. This typically includes updating instance types, AMI IDs, ASG constructs to support multi-architecture (see [Amazon EC2 ASG support for multiple Launch Templates](https://aws.amazon.com/about-aws/whats-new/2020/11/amazon-ec2-auto-scaling-announces-support-for-multiple-launch-templates-for-auto-scaling-groups/)), and finally deploying or redeploying your infrastructure.
+Now you have a tested and performant application, its time to deploy Arm64 application to your infrastructure.
+If you are running applications on your own on-premise infrastructure, it's your choice to either apply the application by container orchestration like Kubernetes in a Ampere processor based cluster, or using Systemd to manage your application on a single Linux server.
```
Comment (dneary) [inline transition-guide.md]: I am unsure where systemd comes into it - I would talk about updating Terraform, Ansible, Pulumi, etc code to use the correct instance types for creating Arm64 clusters or instances, and deploying or redeploying to a new architecture.

**Review by dneary — CHANGES_REQUESTED** [review]
Selected comments - mostly I think the "Infrastructure as Code" may need to be modified and I would like to suggest merging the first paragraph with your proposed text - otherwise, there are some spelling/grammar fixes I will update post commit, and it looks good! Thank you.

File: transition-guide.md (hunk 1 above)
Comment (binh-ampere) [reply transition-guide.md]: sure, it has been reverted to initial statement.

File: transition-guide.md (hunk 2 above)
Comment (binh-ampere) [reply transition-guide.md:104]: reverted

File: transition-guide.md (hunk 3 above)
Comment (binh-ampere) [reply transition-guide.md]: Done, merged to one line.

File: transition-guide.md (hunk 4 above)
Comment (binh-ampere) [reply transition-guide.md]: I was thinking of a standalone condition. But you are right, users can manage bare-metal or local cluster by infrastructure as code tool as well. Updated per your suggestion, thanks.

**Review by dneary — APPROVED** [review]
Approved! Ready to commit.

**Pull request merged**